src/demos/calcite-label.html

```diff
@@ -82,6 +82,13 @@ <h3>small wrapping calcite-input</h3>
         <calcite-input></calcite-input>
       </calcite-label>
 
+      <h3>only render id on parent calcite-label</h3>
```

Comment:
The e2e test should cover this, so this is not needed.

Reply:
Removed 👍